Makefile

```diff
@@ -19,24 +19,24 @@ test_bashscripts: build_bats
 build:
 	docker build -t navigator-admin-backend .
 
-build_local:
+build_dev:
 	docker compose build
 
 unit_test:
-	- docker compose run --rm navigator-admin-backend pytest -vvv unit_tests
+	docker compose run --rm navigator-admin-backend pytest -vvv unit_tests
 
 integration_test:
-	- docker compose run --rm navigator-admin-backend pytest -vvv integration_tests
+	docker compose run --rm navigator-admin-backend pytest -vvv integration_tests
 
 test: unit_test integration_test
 
 migrations:
-	- docker compose run --rm navigator-admin-backend python3 app/initial_data.py
+	docker compose run --rm navigator-admin-backend python3 app/initial_data.py
 
 run: 
-	- docker-compose -f docker-compose.yml up -d --remove-orphans
+	docker-compose -f docker-compose.yml up -d --remove-orphans
 
 start: build run migrations
 
 show_logs: 
-	- docker-compose logs -f
\ No newline at end of file
+	docker-compose logs -f
```

Comment:
We probably want to keep the hyphen on this - as showing the logs is not a gating action

Reply:
Okie dokie, pushing now:)